pages/quantum.js

```diff
@@ -65,596 +65,660 @@ export default function Quantum() {
-            small={4}
-            xmall={2}
-            className=" RoundedEdges  PaddingLeftRight"
-          >
-            <div className=" PaddingLeftRight Margins">
-              <div className="WhiteBackground  RoundedEdges   PaddingLeftRight">
-                <h4 className="TextFormattingH4">Container</h4>
-                <h6 className="TextFormattingH6">
-                  Containers wrap around contents and can be used in different
-                  ways.
-                </h6>
-                <Container>This is a container.</Container>
-              </div>
-              <div className="WhiteBackground  RoundedEdges   Margins PaddingLeftRight">
-                <h4 className="TextFormattingH4">Hide</h4>
-                <h6 className="TextFormattingH6">
-                  Hide will suppress an element in small devices.
-                </h6>
-                <Hide xsmall className="Margins PaddingLeftRight">
-                  <Row>
-                    <Col className="Borders">Reduce the screen size</Col>
-                    <Col className="Borders">to see Hide working</Col>
-                  </Row>
-                </Hide>
-              </div>
-              <div className="WhiteBackground  RoundedEdges   Margins PaddingLeftRight">
-                <h4 className="TextFormattingH4">Link</h4>
-                <h6 className="TextFormattingH6">
-                  Link styled according to Catho&apos;s Style Guide. Makes it
-                  easier to style an anchor tag.
-                </h6>
-                <Link href="https://www.catho.com.br">Link</Link>
-              </div>
-            </div>
-          </Col>
-          <Col
-            large={6}
-            medium={4}
-            small={4}
-            xmall={2}
-            className=" RoundedEdges  PaddingLeftRight"
-          >
-            <div className=" PaddingLeftRight Margins">
-              <div className="WhiteBackground  RoundedEdges   PaddingLeftRight">
-                <div>
-                  <h4 className="TextFormattingH4">Rows and Colums</h4>
+      <main>
+        <section>
+          <Container className="Margins ColoredBackground PaddingTopBotttom ">
+            <Row>
+              <article>
+                <Col className=" RoundedEdges  PaddingLeftRight">
+                  <div className=" PaddingLeftRight Margins">
+                    <div className="WhiteBackground  RoundedEdges   PaddingLeftRight">
+                      <h4 className="TextFormattingH4 ">Breadcrumbs</h4>
+                      <h6 className="TextFormattingH6 ">
+                        Breadcrumbs creates a trail of links to help the user to
+                        locate the current page within the website&apos;s
+                        hierarchy.
+                      </h6>
+                      <Breadcrumbs items={breadcrumbsMockItems} />
+                    </div>
+                  </div>
+                </Col>
+              </article>
```

Comment:
```suggestion
              <Col className=" RoundedEdges  PaddingLeftRight">
                <article>
                  <div className=" PaddingLeftRight Margins">
                    <div className="WhiteBackground  RoundedEdges   PaddingLeftRight">
                      <h4 className="TextFormattingH4 ">Breadcrumbs</h4>
                      <h6 className="TextFormattingH6 ">
                        Breadcrumbs creates a trail of links to help the user to
                        locate the current page within the website&apos;s
                        hierarchy.
                      </h6>
                      <Breadcrumbs items={breadcrumbsMockItems} />
                    </div>
                  </div>
                </article>
              </Col>
```
What do you think about putting article inside each col to avoid styling problems?